test/api/tutorials_test.rb

```diff
@@ -54,26 +43,15 @@ def test_tutorials_post
 
     # Check there is a new tutorial
     assert_equal Tutorial.all.length, number_of_tutorials + 1
-    assert_tutorial_model_response last_response_body, tutorial
+    assert_tutorial_model_response last_response_body, data_to_post[:tutorial]
   end
 
   #2: Testing for failure due to incorrect auth token
   # POST /api/tutorials
   def test_tutorial_post_incorrect_auth_token
 
-    tutorial = {
-      unit_id: '1',
-      tutor_id: User.first.id,
-      campus_id: Campus.first.id,
-      capacity: 10,
-      abbreviation: 'LA011',
-      meeting_location: 'LAB34',
-      meeting_day: 'Tuesday',
-      meeting_time: '18:00'
-    }
-
     data_to_post = {
-      tutorial: tutorial,
+      tutorial: FactoryGirl.build(:tutorial),
```

Comment:
You will want to pass in the related unit and other details otherwise this will create new units etc.

Reply:
There are multiple places this applies